common/rpc/interceptor/telemetry.go

```diff
@@ -404,8 +405,13 @@ func (ti *TelemetryInterceptor) handleError(
 				codes.Unauthenticated,
 				codes.NotFound:
 				return
+			case codes.Internal,
+				codes.Unknown:
+				errorHash := common.ErrorHash(err)
+				logTags = append(logTags, tag.NewStringTag("hash", errorHash))
```

Comment:
I like the idea of adding tag with hash even when message is not masked. It can help with log browsing.

Reply:
not sure how valuable it is. For common errors its a duplicate of a code (if we don't have code as a tag we probably should add it). For errors that has dynamic data it is not that userfull.